### DISCUSSION TITLE
NumberKeyboard 可以和本地的只能显示一个吗

### DISCUSSION THREAD

**Discussion by Julycc41** [ant-design/ant-design-mobile#5779] [Q&A]
需求是，输入身份证input.点击的时候显示了NumberKeyboard组件
但是当我点击其他输入框。例如输入名称的时候 
NumberKeyboard有办法关闭 监听失去焦点关闭吗 ，不然此时是有两个键盘的

**Comment by miracles1919** [comment]
NumberKeyboard 是否展示是通过 `visible` 字段，你可以在点击其他输入框的时候设置 `visible=false`
